src/Symfony/Component/Serializer/Tests/Normalizer/PropertyNormalizerTest.php

```diff
@@ -451,19 +451,19 @@ public function testMultiDimensionObject()
     {
         $normalizer = $this->getDenormalizerForTypeEnforcement();
         $root = $normalizer->denormalize([
```

Comment:
I'd rather put the opening bracket on the next line

Reply:
manually adjusted